ramls/item.json

```diff
@@ -51,6 +51,11 @@
       "type": "string",
       "description": "Identifies the source of the call number, e.g., LCC, Dewey, NLM, etc."
     },
+    "effectiveCallNumber": {
```

Comment:
@mreno-EBSCO Raised the question yesterday in refinement. 

Do we want to create some form of grouping object for all of the effective / derived call number parts?

If we do, what would it be called?

Reply:
@mreno-EBSCO @bohdan-suprun @jsavell Do we want to try to group the derived call number properties together in an object or keep them separate at the top level of the item?

Reply:
I'd prefer to have them grouped.

Reply:
@mreno-EBSCO @bohdan-suprun @marc-johnson What should this grouped object look like?